### DISCUSSION TITLE
Training at 768px and 896px seems to yield clones

### DISCUSSION THREAD

**Discussion by SAC020** [TheLastBen/fast-stable-diffusion#1119] [Q&A]
As the subject says, training SD1.5 at these resolutions tends to generate 2 characters instead of one (i.e. "a woman" would generate two quasi-cloned women). I've tried with and without hires fix.

Anyone seen similar results? Any idea how to fix?

TY

**Comment by TheLastBen** [comment]
V1.5 is trained on 512, so you might face that issue even if you finetune it on 768px pics, increase the number of unet steps to help with that.

**Reply by SAC020** [reply]
Thanks.

What was the default learning rate before the latest update? 1e-6 seems really slow, compared to previous script

**Reply by TheLastBen** [reply]
the default just like before is 2e-6, set it higher (3e-6+)for larger datasets
